src/import/protobuf-c/library/protobuf-c.c

```diff
@@ -25,6 +25,9 @@
  * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * -----
+ * 1 line (1869) added by Xively to fix a build warning
```

Comment:
this is too fragile an edit will invalidate the line number

Reply:
Fixed